### PR TITLE
Add WNP 116 for EU (Fixes #13383)

### LINKED ISSUE #13383: WNP 116 EU (PiP)
## Description

Request is for a localized version of the PiP page here: https://www-dev.allizom.org/en-US/firefox/113.0.2/whatsnew/?geo=na

Copy doc: https://docs.google.com/document/d/1SMT3tPBQdGkDdJwJgHnnB_nTWUIPj4mFeZPGUjN94n4/edit?usp=sharing

---

#### Success Criteria
- [ ] Live before 116 release date on August 1st.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNP 116 for EU promoting Picture-in-Picture. Repurposes [existing design / template](https://www.allizom.org/en-US/firefox/113.0/whatsnew/?geo=us) from 113 NA version.

Note: the DE and FR pages feature English voiced videos, but captions are enabled which will display in the language that the user's web browser is set to.

## Issue / Bugzilla link

#13383

## Testing

- http://localhost:8000/en-GB/firefox/116.0/whatsnew/
- http://localhost:8000/de/firefox/116.0/whatsnew/
- http://localhost:8000/fr/firefox/116.0/whatsnew/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx116-fr.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-md mzp-t-content-nospace">
+    <div class="mzp-c-video">
+      <a class="video-play js-video-play" href="https://youtu.be/_obdx3VUzuQ" data-id="_obdx3VUzuQ" data-video-title="Mozilla: The Big Picture" title="Play the video">
+        <img src="{{ static('img/firefox/whatsnew/whatsnew116-eu/video-thumbnail.jpg') }}" alt="" width="1024" height="576">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx116-fr.html:26]: TODO: this thumbnail needs swapping out for an FR specific one.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx116-fr.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    </div>
+
+    <h2 class="wnp-main-title">Regardez des vidéos en pop-up</h2>
+    <p class="wnp-main-tagline has-pip-icon">
+      Épinglez une vidéo dans la fenêtre de votre navigateur grâce à notre nouvelle fonction "Incrustation vidéo". Essayez avec la vidéo ci-dessus. Appuyez sur "lecture" et cliquez sur <strong>the Picture-in-Picture button</strong> pour faire sortir la vidéo.
+    </p>
+
+    <p class="wnp-main-cta">
+      <a class="mzp-c-button mzp-t-product" href="{{ url('firefox.set-as-default.thanks') }}" data-cta-type="button" data-cta-text="Set Firefox as your default browser">
+        Set Firefox as your default browser
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx116-fr.html:37]: TODO: line still needs FR translation

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx116-fr.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  <div class="mzp-l-content mzp-t-content-md mzp-t-content-nospace">
+    <div class="mzp-c-video">
+      <a class="video-play js-video-play" href="https://youtu.be/_obdx3VUzuQ" data-id="_obdx3VUzuQ" data-video-title="Mozilla: The Big Picture" title="Play the video">
+        <img src="{{ static('img/firefox/whatsnew/whatsnew116-eu/video-thumbnail.jpg') }}" alt="" width="1024" height="576">
+      </a>
+    </div>
+
+    <h2 class="wnp-main-title">Regardez des vidéos en pop-up</h2>
+    <p class="wnp-main-tagline has-pip-icon">
+      Épinglez une vidéo dans la fenêtre de votre navigateur grâce à notre nouvelle fonction "Incrustation vidéo". Essayez avec la vidéo ci-dessus. Appuyez sur "lecture" et cliquez sur <strong>the Picture-in-Picture button</strong> pour faire sortir la vidéo.
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx116-fr.html:32]: TODO: `<strong>the Picture-in-Picture button</strong>` still needs FR translation.

**Review by craigcook — APPROVED** [review]
📺

**Comment by flodolo** [comment]
There was a bug report for typos in the German version. Was there an accessible source, e.g. a document, for the translations?
https://bugzilla.mozilla.org/show_bug.cgi?id=1846578

**Comment by craigcook** [comment]
> There was a bug report for typos in the German version. Was there an accessible source, e.g. a document, for the translations? https://bugzilla.mozilla.org/show_bug.cgi?id=1846578

Copy was provided in this doc: https://docs.google.com/document/d/1SMT3tPBQdGkDdJwJgHnnB_nTWUIPj4mFeZPGUjN94n4/

**Pull request merged**